Majasdarbs/04-majasdarbs/Gvido-CV-responsive/css/styles.css

```diff
@@ -0,0 +1,201 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    background-color: rgb(0, 0, 0);
+    color: white;
+    font-family: 'Roboto', 'Arial', 'sans-serif'; 
+    font-size: 16px;
+}
+
+/* Navigation bar >>>>>>>>>>>>>>>>>>>>>*/
+.nav-bar {
+    display: flex;
+    justify-content: center;
+    background-color: white;
+    font-size: 16px;
+    position: fixed;
+    width: 100vw;
+    top: 0;
+}
+
+.nav-bar a{
+    text-decoration: none;
+    color: black;
+    padding: 0.3rem;
+    margin-left: 10px;
+}
+
+.nav-bar a:hover {
+    color: rgb(255, 255, 255);
+    padding: 1rem;
+    background-color: rgb(0, 0, 0);
+    transition: 0.7s;
+}
+
+/* Header >>>>>>>>>>>>>>>>>*/
+
+/* ABOUT ME */
+#about-me {
+    margin-top: 2rem;
+    padding-top: 2rem;
+    padding-bottom: 2rem;
+    background-image: url(../img/bg-pic.jpg);
```

Comment:
probably you can add 
background-size: cover; <- so background always covers available screen size
background-repeat: no-repeat; <- so background doesn't repeat on larger screen sizes